code/modules/arcade/claw_game.dm

```diff
@@ -66,7 +66,14 @@ GLOBAL_VAR(claw_game_html)
 	for(var/i in 1 to img_resources.len)
 		user << browse_rsc(img_resources[i])
 	var/my_game_html = replacetext(GLOB.claw_game_html, "/* ref src */", UID())
-	user << browse(my_game_html, "window=[window_name];size=915x600;can_resize=0")
+	var/datum/browser/popup = new(user, window_name, name, 915, 700, src)
+	popup.set_content(my_game_html)
+	popup.add_stylesheet("page.css", 'code/modules/arcade/page.css')
+	popup.add_stylesheet("Button.scss", 'tgui/packages/tgui/styles/components/Button.scss')
+	popup.add_script("jquery-1.8.2.min.js", 'html/browser/jquery-1.8.2.min.js')
+	popup.add_script("jquery-ui-1.8.24.custom.min.js", 'html/browser/jquery-ui-1.8.24.custom.min.js')
```

Comment:
This pain me on so many level

Reply:
this hurts me as much as it hurts you, this was supposed to be a simple 1 line fix with just importing jquery as a script element but the claw_game.dm loads in before the crane.html and ignores the jquery library the Html is referring to because why would byond allow simple things. Not the ideal solution but this was the only way that I've found that loads the library, if you have any suggestions on changes I am more than happy to try and replace these lines.